benchmarks/gbench/mhp/mhp-bench.cpp

```diff
@@ -54,6 +54,20 @@ int main(int argc, char *argv[]) {
   comm_rank = rank;
   ranks = size;
 
+  // substitute NNN to be rank in output file
+  for (int i = 0; i < argc; ++i) {
+    auto param = std::string(argv[i]);
+    if (param.starts_with("--benchmark_out=")) {
+      const auto nnnPos = param.find("NNN");
```

Comment:
This was the recommendation to handle multiple processes: https://github.com/oneapi-src/distributed-ranges/blob/3a7e6973fa2ec185e12001d498daa416b07b9315/benchmarks/gbench/mhp/mhp-bench.cpp#L126
Does it still try to write the output file?

Reply:
It is good to have all the ranks, at least for reference. The null reporter stops the console output from the other ranks, which is needed.  The MPI recommendation was to look at the time across all ranks and take the maximum, but that did not seem necessary because we have barriers.

Reply:
Cool. So this is exactly what is implemented here. Jsons are available for all ranks, but plots use just rank 0 numbers.

Reply:
Yes. If we see strange numbers, we can check if one of the ranks was slower than the rest.